pyproject.toml

```diff
@@ -24,6 +24,7 @@ pylint = "^2.13.7"
 isort = "^5.10.1"
 black = "^22.3.0"
 pre-commit = "^2.18.1"
+ipykernel = "^6.15.1"
```

Comment:
useful?

Reply:
Not necessary, I removed it from the file